### DISCUSSION TITLE
LABEL in system.conf

### DISCUSSION THREAD

**Discussion by eraserix** [rauc/rauc#847] [Q&A] [answered]
Hi,

Our targets use SSD or NVMe drives, depending on their age. Is it possible to use a partition label/uuid for the device= entry in system.conf? Or do I have to generate the file at some point with the correct device nodes (e.g. /dev/sdaX vs /dev/nvmen1X)

Christoph

**Comment by ejoerns** [comment, marked as answer]
If your system uses udev, this should generate symlinks for label, uuid, etc. You can use these for your `device` entry, as:

    device=/dev/disk/by-uuid/04fafede...

If you want to support both variants with a single system.conf, you can also generate you own symlinks at boot time.
